python/ngraph/ops.py

```diff
@@ -885,38 +1057,6 @@ def broadcast_to(node, new_shape, axis=None, name=None):
 
 
 @nameable_op
-def fake_quantize(data, input_low, input_high, output_low, output_high, levels, name=None):
```

Comment:
I don't think you meant to remove `fake_quantize` with this PR.

Reply:
Right. I restored it.